cns/NetworkContainerContract.go

```diff
@@ -220,3 +227,29 @@ type UnpublishNetworkContainerResponse struct {
 	UnpublishStatusCode   int
 	UnpublishResponseBody []byte
 }
+
+// Validate - Validates network container request policies
+func (networkContainerRequestPolicy *NetworkContainerRequestPolicies) Validate() error {
+
+	// validate ACL policy
+	if strings.EqualFold(networkContainerRequestPolicy.Type, "ACLPolicy") {
+
+		var requestedAclPolicy hcn.AclPolicySetting
+
+		if err := json.Unmarshal(networkContainerRequestPolicy.Settings, &requestedAclPolicy); err != nil {
+
+			return fmt.Errorf("ACL policy failed to pass validation with error: %+v ", err)
+		}
+
+		if requestedAclPolicy.Action == "" {
```

Comment:
Can you strip the spaces in the string and then check if the len is 0 ?

Reply:
Changed.